configure.ac

```diff
@@ -11,6 +11,8 @@ AC_CANONICAL_HOST
 # Checks for programs.
 AC_PROG_CC
 AC_PROG_CPP
+gl_EARLY
+gl_INIT
```

Comment:
The way I read the manual (https://www.gnu.org/software/gnulib/manual/gnulib.html#Initial-import) I think the gl_INIT should be a bit farther down.  It must "come after other checks which may affect compiler invocation".  I'm not sure if AC_PROG_INSTALL or AC_PROG_LIBTOOL fall into that category, but I'd like to do it as in my patch, to call it out.